### ISSUE TITLE
[User Story] - notation verify - remote artifacts

### ISSUE THREAD

**Issue by iamsamirzon** [notaryproject/roadmap#41]
As a verifier, I want to verify remote artifact integrity with remote signatures using my local trust policy and store. 

Note: verify with local key, verify remote artifacts.  Verify local artifacts out of scope RC-1.

Implementation items to complete story:
- [x] https://github.com/notaryproject/notation/issues/228
  - [x] https://github.com/notaryproject/notation/issues/216

**Comment by iamsamirzon** [comment]
@dtzar  = This implementation work is blocked on the CLI spec ( https://github.com/notaryproject/roadmap/issues/36) being finalized for RC-1
@shizhMSFT  - Assigning this to you to align with the allocation in the spreadsheet.

**Comment by yizha1** [comment]
Covered by issue #38 as well

**Comment by iamsamirzon** [comment]
@yizha1  - #38 is related, but we need to separately track them. This item focusses on the experience of using the "Verify" workflow/command.  The #38 is for how we configure the pre-requisite of trust store and trust policy.

cc: @vaninrao10

**Comment by yizha1** [comment]
Close as completed.
